scripts/send-metrics.py

```diff
@@ -131,6 +131,35 @@ def send_metrics_to_segment(event_properties, config):
             traceback.print_exc()
 
 
+def read_git_state():
+    """
+    Return additional, git-related attributes to be merged into event
+    properties.
+    """
+    # %cI gets the committer timestamp, rather than the author
+    # timestamp; the latter could be older when commits have been
+    # rewritten, and the former is more likely to be of interest when
+    # looking at repo checkout age.
+    p = subprocess.run(
+        ['git', 'show', '--no-patch', '--pretty=format:%cI|%D'],
+        capture_output=True, check=True, timeout=5
+    )
+    timestamp, reflist = p.stdout.decode().split('|', 2)
+    # Returns true if master is currently checked out. Does not return
+    # true in the following similar situations:
```

Comment:
How about something like the following?:
```suggestion
    # Returns true if master is currently checked out. Returns false
    # otherwise, which includes the following situations that are similar to,
    # but different from a master checkout:
```